solarkraft/package.json

```diff
@@ -41,8 +41,8 @@
     "test": "mocha --loader=ts-node/esm --exclude test/e2e/**/*.test.ts --exclude test/integration/**/*.test.ts test/**/*.test.ts"
   },
   "dependencies": {
-    "@stellar/stellar-base": "^11.0.1",
-    "@stellar/stellar-sdk": "^12.2.0",
+    "@stellar/stellar-base": "^12.1.1",
```

Comment:
We have to nuke stellar-base at some point. 
The docs explicitly say you shouldn't use both -sdk and -base

Reply:
Good point. It is actually not needed